tests/basictests.cpp

```diff
@@ -522,12 +525,16 @@ namespace parse_api_tests {
     return parser_moving_parser() &&
            parser_parse() &&
            parser_parse_many() &&
+#ifdef SIMDJSON_ENABLE_DEPRECATED_API
```

Comment:
Goodness, we *deeply* need to use a real test framework :)

Reply:
Yes....

Reply:
Just a suggestion, but [doctest](https://github.com/onqtam/doctest) is pretty good!